src/e2e-test/features/gcs/source/GCSToGCSAdditonalTests.feature

```diff
@@ -0,0 +1,87 @@
+@GCS_Source
+Feature: GCS source - Verification of GCS to GCS Additional Tests successful
+
+  @GCS_AVRO_FILE @GCS_SINK_TEST
+  Scenario: To verify data is getting transferred from GCS to GCS using Avro and Json file format
```

Comment:
mention the scenario as its datatypes related as well

Reply:
Done.